Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/local/FederalStateToPackageId.kt

```diff
@@ -16,5 +16,10 @@ enum class FederalStateToPackageId(val packageId: Int) {
     SN(6),
     ST(6),
     SH(4),
-    TH(6)
+    TH(6);
```

Comment:
Is it intentional that some states have the same ID?

Reply:
AFAIK one package can contain multiple states, there isn't one package per state.

Reply:
yep, the stats packages contain multiple states, this is just here to resolve which package maps to which states https://github.com/corona-warn-app/cwa-app-tech-spec/pull/61/files#diff-b934511166d6db28e5cc528bcfc6df5ac63dd27eb1a9cc9192a61f2b16f4b73eR140-R157